src/lib/config.js

```diff
@@ -5,6 +5,7 @@ const { prompt } = require('enquirer')
 const toSentence = require('./to-sentence')
 const util = require('util')
 const mkdirp = util.promisify(require('mkdirp'))
+const isHostname = require('./is-hostname')
```

Comment:
`is-hostname` is quite a generic name for this module, since it's only doing Tako validation. i think it's unlikely it'll be used outside of `config.js`, so i'd consider inlining it. you'd have to shuffle the tests around then though